partcad/src/partcad/project_factory_git.py

```
@@ -124,9 +127,13 @@ def _clone_or_update_repo(self, repo_url, cache_dir=None):
                             before = repo.active_branch.commit
                             origin = repo.remote("origin")
                             # Need to check for updates
-                            origin.fetch()
+                            origin.fetch(env={
+                                "GIT_CONFIG_PARAMETERS": git_config_params
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)